exercises/resistor-color-duo/package.yaml

```diff
@@ -1,5 +1,5 @@
 name: resistor-color-duo
-version: 2.1.0.2
+version: 0.1.0.3
```

Comment:
I would actually be okay with 2.1.0.3, and furthermore I think 2.1.0.3 is a better choice.

The only specification we have in https://github.com/exercism/haskell/blob/master/README.md#exercise-versioning is that "Exercises based on a `canonical-data.json` [...] should use its version plus a serial number.", so this all hinges on what the phrase *based on* means. I would accept "uses the problem-specifications tests, minus one of them" as acceptable for *based on*. We have also accepted "uses the problem-specifications tests, plus a few extras" as acceptable for *based on*, with an example being https://github.com/exercism/haskell/blob/master/exercises/difference-of-squares/test/Tests.hs#L29 and https://github.com/exercism/haskell/blob/master/exercises/difference-of-squares/package.yaml#L2. If we need to write in the README some rules for what is an acceptable standard for "based on", I'm all good with that.

My reason for why 2.1.0.3 is better: if there is ever a change 2.2.0 that adds a two-colour test case, we would likely want to add it to the track, and still be able to. So let's leave the version here so that tools can know whether this has happened. If we use 0.1.0.x I could certainly add it to the list of ignored exercises in https://github.com/petertseng/exercism-problem-specifications/blob/up-to-date/up-to-date/hs.rb#L3-L9, but then that script will just be blind to problem-specifications changes to resistor-color-duo.

For an example of where we specifically chose to ignore a problem-specifications file, see https://github.com/exercism/haskell/blob/master/exercises/triangle/test/Tests.hs whose version is 0.1.0.4 in https://github.com/exercism/haskell/blob/master/exercises/triangle/package.yaml#L2 , which is indeed not anything like https://github.com/exercism/problem-specifications/blob/up-to-date/exercises/triangle/canonical-data.json.

Reply:
> that script will just be blind to problem-specifications changes

I agree with this sentiment, and had it moments before coming this far in your review.